infrastructure/charts/streamlit/values.yaml

```diff
@@ -0,0 +1,106 @@
+# Default values for streamlit.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: registry.datexis.com/s40781/streamlit
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: latest
+
+env:
+  - name: POSTGRES_SCRAPING_HOST
+    value: postgresql
+  - name: POSTGRES_SCRAPING_PORT
+    value: "5432"
+  - name: POSTGRES_SCRAPING_USER
+    valueFrom:
+      secretKeyRef:
+        name: scraping-secret
+        key: postgres-user
+  - name: POSTGRES_SCRAPING_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: scraping-secret
+        key: postgres-password
+  - name: POSTGRES_GREEN_DB_HOST
+    value: postgresql
+  - name: POSTGRES_GREEN_DB_PORT
+    value: "5432"
+  - name: POSTGRES_GREEN_DB_USER
+    valueFrom:
+      secretKeyRef:
+        name: green-db-secret
+        key: postgres-user
+  - name: POSTGRES_GREEN_DB_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: green-db-secret
+        key: postgres-password
+
+imagePullSecrets:
+   - name: private-registry-auth
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 8501
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
```

Comment:
not necessary

Reply:
Removed [16c58f7](https://github.com/calgo-lab/green-db/pull/91/commits/16c58f74267c8c88732b4aa55a673cf4d37c6fc0)